packages/core/src/Stores/ui-store.js

```diff
@@ -227,6 +227,24 @@ export default class UIStore extends BaseStore {
         return !!this.account_switcher_disabled_message;
     }
 
+    @action.bound
+    filterNotificationMessages() {
+        this.notifications = this.notification_messages.filter(notification => {
+            if (notification.platform === undefined) {
+                return true;
+            } else if (notification.platform && notification.platform.includes(getPathname())) {
+                return true;
+            } else if (notification.platform && !notification.platform.includes(getPathname())) {
+                if (notification.is_disposable) {
+                    this.removeNotificationMessage({ key: notification.key });
+                    this.removeNotificationByKey({ key: notification.key });
+                }
+                return false;
```

Comment:
You could leave out this `return false;` at it'll fall to the `return false;` at line 244.

Reply:
You're right. Thank you!